### ISSUE TITLE
Add ability to download PDF of entire website/sections

### ISSUE THREAD

**Issue by win7guru** [w3c/wai-wcag-quickref#272]
The website in question is the one found here

[WCAG22 Site](https://www.w3.org/WAI/WCAG22/quickref)

An example of this is how Web AIM offers a PDF found here 

[Web AIM WCAG2 Checklist](https://webaim.org/standards/wcag/WCAG2Checklist.pdf) for the accompanying site 

[Web AIM Site](https://webaim.org/standards/wcag/checklist#sc1.3.1)

The following options could be made available for your site

- [ ] Ability to download a PDF containing all sections
- [ ] Ability to download a PDF containing selected sections

Please add this if possible.

**Comment by iadawn** [comment]
What problem do you see this addressing?

It is possible to print as a PDF and have a PDF version of the QuickRef information.

**Comment by win7guru** [comment]
> What problem do you see this addressing?
> 
> It is possible to print as a PDF and have a PDF version of the QuickRef information.

It is not a problem, rather an enhancement to accessibility.  It takes longer to print the PDF version if you have printer access on your phone.  I would much rather have a quick one tap solution.  It is an accessibility site, after all.

Perhaps I should have labeled it enhancement instead.  Can you implement this feature or a print button overlay?
